cdc/sink/table_sink.go

```diff
@@ -55,6 +57,15 @@ func (t *tableSink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) error
 // is required to be no more than global resolvedTs, table barrierTs and table
 // redo log watermarkTs.
 func (t *tableSink) FlushRowChangedEvents(ctx context.Context, resolvedTs uint64) (uint64, error) {
+	logAbnormalCheckpoint := func(ckpt uint64) {
+		if ckpt > resolvedTs {
+			log.L().WithOptions(zap.AddCallerSkip(1)).
+				Warn("checkpoint ts > resolved ts, flushed more than emitted",
```

Comment:
If this happens, is it better to return an error and pause the changefeed

Reply:
The probability is high after table rescheduling, return error may cause changefeed stuck easily. 
In addition, this situation does not necessarily mean data loss.